src/sentry/integrations/slack/utils/users.py

```diff
@@ -74,3 +78,55 @@ def get_slack_data_by_user(
                 slack_data_by_user[user] = slack_info_by_email[email]
                 break
     return slack_data_by_user
+
+
+# USING SDK CLIENT
+
+
+def format_slack_data_by_user(emails_by_user, slack_info_by_email):
+    slack_data_by_user: MutableMapping[User, Mapping[str, str]] = {}
+    for user, emails in emails_by_user.items():
+        for email in emails:
+            if email in slack_info_by_email:
+                slack_data_by_user[user] = slack_info_by_email[email]
+                break
```

Comment:
This is a good example of a pattern you can look at and know using functions would be better. Instead of a break, using a return is cleaner and easier to read, and easier to extend as well for the future

Reply:
the break is to break out of the inner for loop?